src/go/configmanager/flags/flags.go

```diff
@@ -76,6 +78,8 @@ var (
   When disabled, config generator will not make external calls to determine the JWKS URI, 
 	but the 'jwks_uri' field must not be empty in any authentication provider. 
 	This should be disabled when the URLs configured by the API Producer cannot be trusted.`)
+	DependencyErrorBehavior = flag.String("dependency_error_behavior", commonpb.DependencyErrorBehavior_BLOCK_INIT_ON_ANY_ERROR.String(),
+		`The behavior all Envoy filter will adhere to when waiting for external dependencies during filter config.`)
```

Comment:
Can we list out all option strings that users can use?

Reply:
Sure, i'll link to the proto so it won't go out of sync.